nestkernel/nest_names.cpp

```diff
@@ -137,15 +142,30 @@ const Name frozen( "frozen" );
 
 const Name g( "g" );
 const Name g_ahp( "g_ahp" );
+const Name g_AMPA( "g_AMPA" );
 const Name g_ex( "g_ex" );
+const Name g_GABAA( "g_GABAA" );
+const Name g_GABAB( "g_GABAB" );
 const Name g_in( "g_in" );
 const Name g_K( "g_K" );
+const Name g_KL( "g_KL" );
 const Name g_Kv1( "g_Kv1" );
 const Name g_Kv3( "g_Kv3" );
 const Name g_L( "g_L" );
 const Name g_Na( "g_Na" );
+const Name g_NaL( "g_NaL" );
+const Name g_NMDA( "g_NMDA" );
 const Name g_rr( "g_rr" );
 const Name g_sfa( "g_sfa" );
+const Name g_spike( "g_spike" );
+const Name GABA_A_E_rev( "GABA_A_E_rev" );
```

Comment:
See related comment for ``AMPA``; also the conductances follow the "standard logic" with ``g_`` coming first, which comforts me in the idea that ``E_rev_GABA_A`` would be better.